node_modules/gh-groups/tests/test-groups.js

```diff
@@ -598,4 +598,155 @@ describe('Groups', function() {
         });
     });
 
+    describe('Getting user memberships', function() {
+
+        /**
+         * Test that verifies getting a user's memberships
+         */
+        it('verify getting user memberships', function(callback) {
+            // Create a group by creating a new organisational unit
+            OrgUnitTestUtil.assertCreateOrgUnit(global.tests.admins.cam2013.client, 'Test org unit', 'course', null, function(orgUnit) {
+
+                // Create a user and add them to the group
+                TestsUtil.generateTestUsers(global.tests.apps.cam2013, 1, false, function(user) {
+                    var update = {};
+                    update[user.profile.id] = true;
+                    GroupsTestUtil.assertUpdateGroupMembers(global.tests.admins.cam2013.client, orgUnit.GroupId, update, function() {
+
+                        // Get the user's memberships
+                        GroupsTestUtil.assertGetUserMemberships(global.tests.admins.cam2013.client, user.profile.id, null, null, [orgUnit.GroupId], function(memberships) {
```

Comment:
Oh I see you pass an array. Can you change the name of this to `assertGetUserMembershipsEquals` ?

Also, `GroupsTestUtil` should be changed to `GroupsTestsUtil` since the folder name is `tests`